.eslintrc.js

```diff
@@ -22,37 +23,45 @@ module.exports = {
 
     'no-console': 'off',
 
-    'import/no-extraneous-dependencies': ['error', {
-      devDependencies: true,
-    }],
-
-    'no-constant-condition': ['error', {
-      checkLoops: false,
-    }],
-
-    'max-len': ['error', 140, 2, {
-      ignoreStrings: true,
-      ignoreTemplateLiterals: true,
-      ignoreTrailingComments: true,
-      ignoreUrls: true,
-    }],
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: true,
+      },
+    ],
 
-    'no-unused-vars': ['error', {
-      vars: 'all',
-      varsIgnorePattern: '_([0-9]+)?$',
-      args: 'after-used',
-      argsIgnorePattern: '_([0-9]+)?$',
-    }],
+    'no-constant-condition': [
+      'error',
+      {
+        checkLoops: false,
+      },
+    ],
 
-    'import/prefer-default-export': 'off',
-    'object-curly-newline': 'off',
-    'arrow-parens': [
+    'max-len': [
+      'error',
+      140,
       2,
-      'as-needed',
       {
-        requireForBlockBody: true,
+        ignoreStrings: true,
+        ignoreTemplateLiterals: true,
+        ignoreTrailingComments: true,
+        ignoreUrls: true,
+      },
+    ],
+
+    'no-unused-vars': [
+      'error',
+      {
+        vars: 'all',
+        varsIgnorePattern: '_([0-9]+)?$',
+        args: 'after-used',
+        argsIgnorePattern: '_([0-9]+)?$',
       },
     ],
+
+    'import/prefer-default-export': 'off',
+    'object-curly-newline': 'off',
+    'arrow-parens': [2, 'always'],
```

Comment:
rewrite this rule to https://eslint.org/docs/rules/arrow-parens#always to match Prettier config